Kitodo/src/main/java/org/kitodo/production/forms/DesktopForm.java

```diff
@@ -99,6 +100,7 @@ public DesktopForm() {
         try {
             if (ServiceManager.getSecurityAccessService().hasAuthorityToViewProcessList() && processList.isEmpty()) {
                 processList = ServiceManager.getProcessService().loadData(0, 10, SORT_ID, SortOrder.DESCENDING, null);
+                Collections.sort(processList, (one, other) -> other.getId() - one.getId());
```

Comment:
Why is this neccessary? The processes are already returned in a sorted list, as indicated by the `SORT_ID` constant passed as the third parameter to `loadData` in the previous line, aren't they?

If you just want to reverse the order and start with the lowest ID, you should use `SortOrder.ASCENDING` instead of `DESCENDING`, I guess.